src/uagents/agent.py

```diff
@@ -159,7 +170,7 @@ def sign_digest(self, digest: bytes) -> str:
 
     def sign_registration(self) -> str:
         return self._identity.sign_registration(
-            self._reg_contract.address, self.get_registration_sequence()
+            self._reg_contract.address, self._get_registration_sequence()
```

Comment:
Made this private because of pylint complaint, but also because I think it should be.